lib/service/database.js

```diff
@@ -284,6 +288,12 @@ DatabaseService.prototype.queueAnalysisOperations = function(analysis, callback)
                 asyncQueries.push({
                     query: updateNodeStatusAtAnalysisCatalogQuery(nodeIds, Node.STATUS.RUNNING)
                 });
+                if (sqlWrappedNode.preCheckNodeQuery() !== undefined) {
+                    asyncQueries.push({
+                        query: readOnlyTransactionQuery([sqlWrappedNode.preCheckNodeQuery()]),
```

Comment:
Is the read only translation something we want to apply in all pre check queries?